zc_plugins/POSM/v6.0.0/admin/products_options_stock_view_all.php

```diff
@@ -345,6 +345,9 @@
                 <tr>
                     <td colspan="<?= STATIC_FIELD_COUNT ?>" class="text-center"><?= POSM_VIEW_ALL_NO_PRODUCTS_TO_LIST ?></td>
                 </tr>
+            </tbody>
+        </table>
+    <?= '</form>' ?>
```

Comment:
this is not what i envisioned the use of the short echo tag.

the short echo tag is to echo php vars within a html block.

good coding standards would remove the short echo and just have the `</form>` within the html.

Reply:
But, since the form is rendered via `zen_draw_form`, the end-form tag should also (IMO) be rendered via PHP.

Reply:
i will never make a comment on formatting again.

while i see your reasoning, i do not agree with it.  it strikes me as something someone else would say in order that their IDE does not complain.